tests/microbenchmarks.py

```diff
@@ -92,6 +96,7 @@ def speak(self):
 
     """), timed=True)
 
+
 def test_function_var_load(test_case):
     print("Running", "test_function_var_load")
     test_case.runAsJava(adjust("""
```

Comment:
At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils